examples/js/gateio-futures.js

```diff
@@ -0,0 +1,74 @@
+const ccxt = require ('../../ccxt');
+
+const exchange = new ccxt.gateio ({
+    'apiKey': 'YOUR_API_KEY',
+    'secret': 'YOUR_SECRET_KEY',
+    'options': {
+        'defaultType': 'future',
+    },
+})
+
+;(async () => {
+    // exchange.setSandboxMode (true)
+
+    const markets = await exchange.loadMarkets ()
+
+    exchange.verbose = true // uncomment for debugging purposes if necessary
+
+    // Example 1: Creating and canceling a linear future (limit) order
+    try {
+        const symbol = 'LTC/USDT:USDT'
```

Comment:
This is a `swap` type symbol, not a `future` type, though: https://github.com/ccxt/ccxt/wiki/Manual#contract-naming-conventions

A future will have a delivery date. A swap is a future that does not have a delivery date.

Reply:
> A swap is a future that does not have a delivery date

@kroitor So technically a swap is also a future with the difference of not having an expiration date, but I can change it to swap no problem. I just did not add a real future because the example would be outdated as soon as the future expired

Reply:
@kroitor Renamed, Is there any other example that you think is missing?

Reply:
> I just did not add a real future because the example would be outdated as soon as the future expired

Instead of hardcoding it, we can also try picking whatever active BTC-week future we have.

Reply:
When we work with a swap market symbol, then the defaultType is also expected to be `swap`.

Reply:
> Instead of hardcoding it, we can also try picking whatever active BTC-week future we have.

@kroitor good idea will do that

> When we work with a swap market symbol, then the defaultType is also expected to be swap.
@kroitor yeah For this particular case it's pretty much the  same because swap+futures are loaded together but I understand that might not be semantically correct so will change it too

Reply:
@kroitor  Done 🙂